content/projects/morse-code/_index.md

```diff
@@ -80,12 +81,10 @@ morseCodeToLetters(".... .. / - .... . .-. .");
 
 Make sure to cater for punctuation marks and all special characters.
 
-Include the following assertions in your code:
+- Include the following assertions in your code:
 
-- Assert that the output and input both have the same number of characters represented. Use the string `the number of input and output characters is not the same` when both input and output characters are not equal.
-- Assert that there are the right number of spaces represented in the output. Use the string `the number of input and output space characters is not the same` when both input and output space characters are not equal.
-
-Make sure that when you do this, your code remains clean and dry.
+  - Assert that the output and input both have the same number of characters represented. Use the string `the number of letter characters is not equal to the number of morse code characters` when both input and output characters are not equal.
+  - Assert that there are the right number of spaces represented in the output. Use the string `the number of letter space characters is not equal to the number of morse code space characters` when both input and output space characters are not equal.
```

Comment:
The current automarker for this project is deactivated with the assertions strings being one of the problems? Any idea on how to make them less weird/confusing.

Check out the related config on the automarker projects repo to see why it was deactivated.

It's fine if you can't come up with anything now.

Reply:
Just tell them to think about what error messages would be useful and then they can write those. Coming up with clear error messages is something a developer needs to be able to do.

Spoonfeeding people does make the code easier to review. But  spoonfeeding is bad for education.

Reply:
> The current automarker for this project is deactivated with the assertions strings being one of the problems? Any idea on how to make them less weird/confusing.
> 
> Check out the related config on the automarker projects repo to see why it was deactivated.
> 
> It's fine if you can't come up with anything now.

Oh I see, cool no problem. I didn't have the whole picture.